images/warp10/static.tokens

```diff
@@ -0,0 +1,9 @@
+token.write.0.name=writeTokenStatic
+token.write.0.producer=42424242-4242-4242-4242-424242424242
```

Comment:
is this a test file or example file? Or does all 42's actually work?

Reply:
I just reused upstream's from here https://github.com/senx/warp10-docker/blob/master/predictible-tokens-for-ci/ci.tokens
AFAIK that's a valid, if highly unlikely, uuid.

Reply:
oh are these just for CI purposes?

Reply:
not only, we use a static token for deployment as well. Different Dockerfile though.